tests/queries/0_stateless/02723_zookeeper_name.reference

```diff
@@ -0,0 +1,4 @@
+Create Tables
+Insert Data
+"t1","default",1
+"t2","default",2
```

Comment:
The test contains a race condition: data parts can be merged before the test finishes. Therefore, the test will be flaky. Let's simplify it.

Reply:
fixed by removing the second `INSERT`